webapp/components/post_view/post_list.jsx

```diff
@@ -210,20 +210,18 @@ export default class PostList extends React.PureComponent {
         }
 
         if (postList && prevPosts && posts && posts[0] && prevPosts[0]) {
-            // A new message was posted, so scroll to bottom if it was from current user
-            // or if user was already scrolled close to bottom
+            // A new message was posted, so scroll to bottom if user
```

Comment:
I think we still want to scroll down if the user was scrolled up, but it was posted in the center channel